proof/invariant-abstract/AARCH64/ArchKHeap_AI.thy

```diff
@@ -9,9 +9,7 @@ theory ArchKHeap_AI
 imports KHeapPre_AI
 begin
 
-(* FIXME AARCH64: if_option is missing all cases for None/Some = if .. *)
-
-declare if_option_Some_eq[simp]
+declare if_option_eq[simp]
```

Comment:
What about the other arches? Do you want them to get this as well? I realise for this week it's a good idea to not do that, but if yes, we shouldn't forget.

Reply:
I tried that in a separate branch (in fact, I tried to enable it globally), but it broke too many existing proofs in annoying ways. So keeping it AARCH64 only is better I think.

We should probably actually consider not making it globally `[simp]` there either and use it only locally to not diverge too much. I could have a look at that separately.

Reply:
That's an excellent question. I do not know which is better. AARCH64 uses a lot of option monad projections, but then again so did RISCV64 and it didn't have that declaration, so I think you're right. Not sure we want it this week, but worth taking a look.

Reply:
I'll run a session in the background and see what breaks